src/workers/newNotificationV2Mail.ts

```diff
@@ -856,6 +857,24 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
       keyword: keyword?.flags?.title || keyword.value,
     };
   },
+  user_gifted_plus: async (con, user) => {
+    const { subscriptionFlags } = await con.getRepository(User).findOneOrFail({
+      where: {
+        id: user.id,
+      },
+      select: ['subscriptionFlags'],
+    });
+    const gifter = await con.getRepository(User).findOneOrFail({
+      where: {
+        id: subscriptionFlags?.gifterId,
+      },
```

Comment:
Watch out, make sure `gifterId` is not `undefined` or `null`, because this will return random user from the DB then. Probably good to throw in that case.